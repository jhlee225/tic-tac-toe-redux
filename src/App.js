import React from "react";
import TicTacToe from "./features/tictactoe/tttContainer";
//Dasd
function App() {
  return (
    <div className="App">
      <header className="App-header"></header>
      <TicTacToe />
    </div>
  );
}

export default App;
