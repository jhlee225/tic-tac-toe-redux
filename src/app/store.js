import { configureStore } from "@reduxjs/toolkit";
import tttReducer from "../features/tictactoe/tttSlice";

export default configureStore({
  reducer: {
    ttt: tttReducer,
  },
});
