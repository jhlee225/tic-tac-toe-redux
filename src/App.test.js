import React from "react";
import { render } from "@testing-library/react";
import { Provider } from "react-redux";
import store from "./app/store";
import App from "./App";

it("Render App", () => {
  const AppRendered = render(
    <Provider store={store}>
      <App />
    </Provider>
  );
  expect(AppRendered.container).toMatchSnapshot();
  expect(AppRendered.container).toBeInTheDocument();
});
