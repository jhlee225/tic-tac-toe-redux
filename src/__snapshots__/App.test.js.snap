// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App 1`] = `
<div>
  <div
    class="App"
  >
    <header
      class="App-header"
    />
    <div
      class="game"
    >
      <div
        class="game-board"
      >
        <div>
          <div
            class="board-row"
          >
            <button
              class="square"
              id="0"
              role="button"
            />
            <button
              class="square"
              id="1"
              role="button"
            />
            <button
              class="square"
              id="2"
              role="button"
            />
          </div>
        </div>
        <div>
          <div
            class="board-row"
          >
            <button
              class="square"
              id="3"
              role="button"
            />
            <button
              class="square"
              id="4"
              role="button"
            />
            <button
              class="square"
              id="5"
              role="button"
            />
          </div>
        </div>
        <div>
          <div
            class="board-row"
          >
            <button
              class="square"
              id="6"
              role="button"
            />
            <button
              class="square"
              id="7"
              role="button"
            />
            <button
              class="square"
              id="8"
              role="button"
            />
          </div>
        </div>
      </div>
      <div
        class="game-info"
      >
        <div>
          <div>
            Next player: X
          </div>
          <label
            class="switch"
          >
            <input
              type="checkbox"
            />
            <span
              class="slider round"
            />
          </label>
        </div>
        <ol>
          <li>
            <button>
              Go to game start
            </button>
            <span>
               
              
               
            </span>
          </li>
        </ol>
      </div>
    </div>
  </div>
</div>
`;

exports[`render App 1`] = `
<div>
  <div
    class="App"
  >
    <header
      class="App-header"
    />
    <div
      class="game"
    >
      <div
        class="game-board"
      >
        <div>
          <div
            class="board-row"
          >
            <button
              class="square"
              id="0"
              role="button"
            />
            <button
              class="square"
              id="1"
              role="button"
            />
            <button
              class="square"
              id="2"
              role="button"
            />
          </div>
        </div>
        <div>
          <div
            class="board-row"
          >
            <button
              class="square"
              id="3"
              role="button"
            />
            <button
              class="square"
              id="4"
              role="button"
            />
            <button
              class="square"
              id="5"
              role="button"
            />
          </div>
        </div>
        <div>
          <div
            class="board-row"
          >
            <button
              class="square"
              id="6"
              role="button"
            />
            <button
              class="square"
              id="7"
              role="button"
            />
            <button
              class="square"
              id="8"
              role="button"
            />
          </div>
        </div>
      </div>
      <div
        class="game-info"
      >
        <div>
          <div>
            Next player: X
          </div>
          <label
            class="switch"
          >
            <input
              type="checkbox"
            />
            <span
              class="slider round"
            />
          </label>
        </div>
        <ol>
          <li>
            <button>
              Go to game start
            </button>
            <span>
               
              
               
            </span>
          </li>
        </ol>
      </div>
    </div>
  </div>
</div>
`;
